Header header

uint8    mode  
uint8    gamma
uint8    flags
float64  range
float64  gain
float64  speed_of_sound
float64  salinity